main.go

```diff
@@ -42,6 +42,14 @@ type flagVal struct {
 	OverWrite_flag bool
 	Instance_flag  string
 	Verbose_flag   bool
+	File_flag      bool
+}
+
+// contains local and server paths
+type pathVal struct {
+	RootWorkingDirectoryLocal  string
+	RootWorkingDirectoryServer string
+	StartingPathServer         string
```

Comment:
So pathVal holds the working directory for both the local and server hosts, as well as the "starting path" for the server. Is that the starting path of an individual file, or something else?